### PR TITLE
added message_controller.js

### PR DESCRIPTION
**Pull request by JuditBP**

### CODE REVIEW

File: app/views/appointments/index.html.erb
Comment (JuditBP) [inline app/views/appointments/index.html.erb:1]: Liam told me, the any is implicit, no needed in this case

File: app/views/layouts/application.html.erb
Comment (JuditBP) [inline app/views/layouts/application.html.erb:1]: I commented out this bootstrap line

**Review by singhshriyam — APPROVED** [review]
great job

File: app/views/layouts/application.html.erb
Comment (JuditBP) [reply app/views/layouts/application.html.erb:1]: Because it caused the footbar movement

**Review by singhshriyam — APPROVED** [review]

**Review by singhshriyam — APPROVED** [review]

**Pull request merged**